bootcamp/tutorials/integration/haystack/full_text_search_with_milvus_and_haystack.ipynb

```diff
@@ -0,0 +1,602 @@
+{
```

Comment:
<p>please use the server address, e.g.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1524/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>